src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -312,10 +312,10 @@ export class QueryEditor extends EditorPane {
 
 		// TODO: Allow query provider to provide the language mode.
 		if (this.input instanceof UntitledQueryEditorInput) {
-			if ((providerId === 'KUSTO') || this.languageService.getExtensions('Kusto').indexOf(fileExtension) > -1) {
+			if ((providerId === 'KUSTO') || this.languageService.getExtensions('kusto').indexOf(fileExtension) > -1) {
```

Comment:
unrelated change, but now getExtensions expects language ID instead of language name.  Likely the provider test would be true in any case, but there could be some new corner case when this test will now fail without this change.